src/components/editor/property/InputListLOC.jsx

```diff
@@ -62,11 +60,20 @@ class InputListLOC extends Component {
   }
 
   render() {
-    if (this.props.lookupConfig?.uri === undefined) {
+    // Don't render if no property template yet
+    if (!this.props.propertyTemplate) {
+      return null
+    }
+
+    if (this.props.lookupConfig?.length > 1) {
+      alert(`There are multiple configured list lookups for ${this.props.propertyTemplate.propertyURI}`)
+    }
```

Comment:
there was a test ostensibly for this, so I put in the guard clause

Reply:
i think you are right that some of these errors were for vestigial code.  In this PR, I decided to explicitly check when there was an undefined uri, and then created the explicit guard clause to match the test that was there for when the lookupConfig json has a property uri in there twice.  

You'll also notice that I had to redo the inline fixture props for the tests, and I decided to name them something helpful to help the readability of the tests.

I didn't delve far into "what, precisely is in the lookupConfig and why should this component get lookupConfig[0] when the other didn't ..." -- so yeah, there might be room for cleaner code there.  Hardcoding to "first one" hidden in the code is not my fav ... but I wanted to get the PR done.